### PR TITLE
Additional, non-lethal Carrion Spiderlings.

### PR DESCRIPTION
**Pull request by Zenitheevee**
## About The Pull Request

This PR Attempts to increase Carrion's non-lethal toolkits by giving them 7 new tools under their belt.. or, spiderlings under their webs?

Narcolepsy Spiderling: This is a Biter type spiderling, much like the healing spiderling. This little thing carries 5 units of Soporific Toxin, Injecting it into its host and putting them effectively to sleep. Great for kidnapping and snagging implant objectives, Bad for randomly activating on people.

Mending Spiderling: Another Biter type, This one's sole use is to mimic the carrion's stasis, But in a way that fully incapacitates the host. This really helps with converting people safely, but be aware they're unable to defend themselves, especially since you need to donate a genome sample to kickstart the process. 

Injector Spiderling: The last Biter of the group, This one is unique in that it doesn't have its own reagents, rather it relies on you giving it your own. Dust off that vagabond's guide to chemistry and make up your own cocktails to buff or hinder.

Cleaner Spiderling: This little fellas got a really shiny back, and for a reason. Upon being activated, It causes a chemical reaction that spawns a wave of cleaning suds. Be careful, they are quite slippery.

Smoker Spiderling: "Y'want lung cancer?" Well, this guys got it. Creates a large puff of smoke for you to vanish in.

Disruptor Spiderling: Miniaturized EMP in a living form. Loves fuzzy socks. Creates a small, weak EMP. 

Pheromone Spiderling: You know its kinda ass to try and lead someone through your maint lair when your spiders attack them, And now you dont have to worry about that. Tags the person as "Friendly" on implant, And activating it removes the "tag" However, it needs a genome sample to recreate the pheromones.

These are... Not the best balanced, But keep in mind both genome and chemicals are fairly abundant resources to good setups.

## Why It's Good For The Game

Adds non-lethal tools to Carrion, allowing more options that are available while still in the theme.

## Changelog
```changelog
add: Adds 7 new spiderlings, each designed for less-than-lethal to even pacifistic effects.
```

### CODE REVIEW

File: zzzz_modular_occulus/code/game/objects/items/weapons/implant/implants/carrion/smoker_spider.dm [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	name = "smoker spider"
+	desc = "A tiny spider with a pearl-white abdomen covered in smooth bubbles."
+	icon = 'zzzz_modular_occulus/icons/obj/carrion_spiders.dmi'
+	icon_state = "spiderling_smoker"
+	spider_price = 10
+	var/datum/effect/effect/system/smoke_spread/bad/smoke
+
+/obj/item/weapon/implant/carrion_spider/smoker/New()
+	..()
+	smoke = new
```
Comment (Michiyamenotehifunana) [inline zzzz_modular_occulus/code/game/objects/items/weapons/implant/implants/carrion/smoker_spider.dm:11]: The smoke effect ought to be explicitly qdel'd once the spiderling itself is qdel'd, but the GC should probably handle that automatically without lagging the server to hell. Probably. Just keep this in mind if things start lagging to hell next time there's a carrion round on the server.

**Review by Michiyamenotehifunana — DISMISSED** [review]
Everything looks to be in order aside from one minor concern. Approved.

File: zzzz_modular_occulus/code/game/objects/items/weapons/implant/implants/carrion/smoker_spider.dm (hunk 1 above)
Comment (Zenitheevee) [reply zzzz_modular_occulus/code/game/objects/items/weapons/implant/implants/carrion/smoker_spider.dm:11]: Does this new patch fix the issue that _Would_ have been?

Comment (Michiyamenotehifunana) [reply zzzz_modular_occulus/code/game/objects/items/weapons/implant/implants/carrion/smoker_spider.dm:11]: It should!

**Review by Michiyamenotehifunana — APPROVED** [review]
Looks about right!

**Pull request merged**